examples/sample-pub/publisher.js

```diff
@@ -309,6 +310,62 @@ function whenDemoReady(callback) {
   }
 }
 
+/**
+ * Returns true if the URL contains valid Google Article Access (GAA) params.
+ * @return {boolean}
+ */
+function isGaa() {
+  // Validate GAA params.
+  const params = getQueryParams();
+  if (!params.gaa_at) {
+    return false;
+  }
+  if (!params.gaa_n) {
+    console.error('SwG Entitlements: The `gaa_n` URL param is missing.');
```

Comment:
Could you add a comment that points to where the URL parameters are defined?

Reply:
We don't have a public document describing these params yet. I added a TODO to link to such a document though!